3-extra/1-count-words.js

```diff
@@ -25,10 +25,19 @@
 
 function countWords(string) {
   const wordCount = {};
-
-  // write code here
-
-  return wordCount;
+  let arrOfWords = string.split(" ");
+  for (let i = 0; i < arrOfWords.length; i++) {
+    if (wordCount[arrOfWords[i]]) {
+      wordCount[arrOfWords[i]]++;
+    } else {
+      wordCount[arrOfWords[i]] = 1;
+    }
+  }
+  if (string === "") {
```

Comment:
this is an `edge case`
and usually we check things like that at the very beginning of our function ;)